src/main.js

```diff
@@ -125,15 +125,32 @@ function removePoster() {
    showSavedPosters();
 }
 
-// extensions
-
+function getRandomIndex(array) {
+  return array[Math.floor(Math.random() * array.length)];
+}
```

Comment:
Great use of a helper function here! Once your projects get a little robust and you start needing access to these functions in multiple files I would recommend adding functions like these to a `helpers.js` or `utils.js`. I have one of those in pretty much every repo at work!